src/main/java/v/hryhoryk/onlinebookstore/controller/BookController.java

```diff
@@ -18,42 +23,92 @@
 import v.hryhoryk.onlinebookstore.dto.CreateBookRequestDto;
 import v.hryhoryk.onlinebookstore.service.BookService;
 
+@Tag(name = "Book store API",
+        description = "Endpoints for managing book store API")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(summary = "Get all books",
+            description = "Returns paginated and sorted view of all books")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200",
+                    description = "Successfully retrieved"),
+            @ApiResponse(responseCode = "404",
+                    description = "Not found - no books are available")
+    })
     @ResponseStatus(HttpStatus.OK)
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
+    public List<BookDto> getAll(Pageable pageable) {
```

Comment:
it may be beneficial to use `@PageableDefault` in case client did not specify pagination parameters